src/test/java/seedu/address/model/task/TaskDescriptionTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.task;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class TaskDescriptionTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new TaskDescription(null));
+    }
+
+    @Test
+    public void isValidDescription() {
+        // null description
+        assertThrows(NullPointerException.class, () -> TaskDescription.isValidDescription(null));
+
+        // invalid descriptions
+        assertFalse(TaskDescription.isValidDescription(" ")); // spaces only
+
+        // valid descriptions
+        assertTrue(TaskDescription.isValidDescription("")); // empty string
```

Comment:
Oh ya, if empty string is valid, then you might want to add parse_optionalFieldsMissing_success to AddTaskCommandParserTest (i hope i typed correctly)